modules/deploy/pages/deployment-option/self-hosted/kubernetes/kubernetes-cluster-requirements.adoc

```diff
@@ -1,70 +1,182 @@
+
+=== Kubernetes version
+
+*Requirement*: Minimum required Kubernetes version: {supported-kubernetes-version}
 
 Make sure to do the following:
 
 . https://kubernetes.io/docs/tasks/tools/[Install kubectl^].
 . https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/[Configure the `kubeconfig` file for your cluster^].
 
-== Helm
+=== Helm version
 
-Minimum required Helm version: {supported-helm-version}
+*Requirement*: Minimum required Helm version: {supported-helm-version}
 
 https://helm.sh/docs/intro/install/[Install Helm^].
 
-== Number of worker nodes
+=== Worker nodes
 
-You must provision one worker node for each Redpanda broker that you plan to deploy in your Redpanda cluster. Each Pod replica that runs a Redpanda broker requires its own dedicated worker node for the following reasons:
+*Requirement*: Provision one dedicated worker node for each Redpanda broker that you plan to deploy in your Redpanda cluster.
+
+Each Pod replica that runs a Redpanda broker requires its own dedicated worker node for the following reasons:
 
 * Redpanda brokers are designed to have access to all resources, such as CPU and memory, on their host machine.
 * External clients access Redpanda brokers using the address of a worker node. See <<External networking>>.
 
-NOTE: The Helm chart configures `podAntiAffinity` rules to make sure that each Redpanda broker runs on its own worker node.
+NOTE: The Redpanda Helm chart configures `podAntiAffinity` rules to make sure that each Redpanda broker runs on its own worker node.
+
+*Recommendation*: <<Deploy at least three Pod replicas>>.
+
+=== CPU
+
+*Requirement*: Two physical, not virtual, cores for each worker node.
+
+*Requirement*: x86_64 (Westmere or newer) and AWS Graviton family processors are supported.
+
+*Recommendation*: Four physical cores are strongly recommended.
+
+*Recommendation*: <<Set resource requests and limits for memory and CPU>>.
+
+=== Memory
+
+*Requirement*: 2 GB or more of memory per core.
+
+=== Storage
+
+*Requirement*: The Redpanda data directory (`/var/lib/redpanda/data`) and the Tiered Storage cache must be mounted on an XFS or ext4 filesystem. For information about supported volume types for different data, see xref:manage:kubernetes/storage/volume-types.adoc[].
+
+CAUTION: Avoid using NFS (Network File System) for the Redpanda data directory or the Tiered Storage cache.
+
+*Recommendation*: Use an XFS filesystem for its enhanced performance with Redpanda workloads.
+
+*Recommendation*: <<kubernetes-volumes, Use local PersistentVolumes that are backed by locally attached NVMe disks>> to store the Redpanda data directory.
+
+*Recommendation*: For setups with multiple disks, use a RAID-0 (striped) array. It boosts speed but lacks redundancy. A disk failure can lead to data loss.
+
+=== Cloud instance types
+
+*Recommendation*: Use instance types that supports local NVMe disks. See xref:./cloud-instance-local-storage.adoc[].
```

Comment:
*Recommendation*: Use instance types that support local NVMe disks. See xref:./cloud-instance-local-storage.adoc[].